.github/workflows/build.yml

```diff
@@ -0,0 +1,22 @@
+name: Build Image
+
+on:
+  pull_request:
+    branches: ["main"]
+  push:
+    branches: ["main"]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read #Needed to build image
+      packages: write #Needed to publish package/image
+
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v3
+
+    - name: Build Image
+      run: |
+            docker build . --file Dockerfile --tag ghcr.io/roboflow/roboflow-react-app:staging
```

Comment:
Can we change the tag back to `roboflow-react-app:latest`?

```docker build . --file Dockerfile --tag roboflow-react-app:latest```

Reply:
Good spot, this has been completed.